pandas/core/arrays/base.py

```diff
@@ -776,7 +786,7 @@ def shift(self, periods: int = 1, fill_value: object = None) -> ExtensionArray:
             b = empty
         return self._concat_same_type([a, b])
 
-    def unique(self):
+    def unique(self) -> ExtensionArray:
```

Comment:
use ExtensionArrayT (return is `self._from_sequence...`)

Reply:
Actually, needs to be `def unique(self: ExtensionArrayT) -> ExtensionArrayT`